src/panels/lovelace/hui-editor.ts

```diff
@@ -36,6 +42,9 @@ class LovelaceFullConfigEditor extends hassLocalizeLitMixin(LitElement) {
             ></paper-icon-button>
             <div main-title>Edit Config</div>
             <paper-button @click="${this._handleSave}">Save</paper-button>
+            <ha-icon class="save-button ${classMap({
+              saved: this._saving! === false,
```

Comment:
I think that the icon should hide again when a user starts typing, indicating that the current work is not saved.